package.json

```diff
@@ -197,12 +197,13 @@
   },
   "resolutions": {
     "vitest>vite": "^4.3.1",
-    "rehype-katex>katex": "^0.16.6"
+    "rehype-katex>katex": "^0.16.6",
+    "pinyin>nodejieba": "^2.6.0"
```

Comment:
nodejieba v2.6.0 is not available on Windows, refer to https://github.com/yanyiwu/nodejieba/issues/205

Reply:
This library is too bad. Two developers previously reported that version 2.6.0 cannot be used.